src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/CcdCallbackType.java

```diff
@@ -0,0 +1,6 @@
+package uk.gov.hmcts.reform.bulkscan.orchestrator.services.ccd;
+
+public enum CcdCallbackType {
```

Comment:
may be name it as `CcdEventType`

Reply:
I don't know, event type might be misleading. This enum represents the kind of action that's happening, rather than a specific CCD event.

Reply:
There is nothing called as callback type :) what you are referring here are purely ccd events attach supplementary evidence/ case creation.

Reply:
This enum is here to provide information about the kind of action we're serving, not which event was triggered. If you're not happy with `CcdCallbackType`, we can come up with something else, but I don't want to create the impression that these are event IDs.